.github/autolabeler.yml

```diff
@@ -0,0 +1,26 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+infra: [".test-infra/*"]
+build: ["buildSrc/*", "release/*", "ownership/*", ".github/*", "gradle/*"]
+website: ["website/*"]
+go: ["sdks/go/*"]
+java: ["sdks/java/*"]
+python: ["sdks/python/*"]
+runner: ["runners/*"]
+examples: ["examples/*", "learning/*"]
```

Comment:
I would keep learning separate, these are different things and commonly mainted by different people who may be interesting in filtering one or the other.